Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.XamlHost/WindowsXamlHostBase.cs

```diff
@@ -0,0 +1,125 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Drawing;
+using System.Security.Permissions;
+using System.Windows.Forms;
+using Microsoft.Toolkit.Win32.UI.XamlHost;
+
+namespace Microsoft.Toolkit.Forms.UI.XamlHost
+{
+    /// <summary>
+    ///     A sample Windows Forms control that hosts XAML content
+    /// </summary>
+    [System.ComponentModel.DesignerCategory("code")]
+    public partial class WindowsXamlHostBase : Control
+    {
+#pragma warning disable SA1401 // Fields must be private
+                              /// <summary>
+                              /// DesktopWindowXamlSource instance
+                              /// </summary>
+        protected Windows.UI.Xaml.Hosting.DesktopWindowXamlSource desktopWindowXamlSource;
```

Comment:
Why can't this be converted to `protected DesktopWindowXamlSource DesktopWindowXamlSource { get; }` so we can comply with design rules and not have to suppress `SA1401`?